pkg/metrics/metrics.jsx

```diff
@@ -641,6 +646,39 @@ class CurrentMetrics extends React.Component {
             cpu_label = this.state.cpuUsed + '%';
         }
 
+        let disksUsage = [];
+        if (this.disksNames.length > 0 && this.samples[14] && this.samples[15]) {
+            disksUsage = this.disksNames.map((name, i) => [
+                name,
+                this.samples[14][i] >= 1 ? cockpit.format_bytes_per_sec(this.samples[14][i]) : "0",
+                this.samples[15][i] >= 1 ? cockpit.format_bytes_per_sec(this.samples[15][i]) : "0",
+            ]);
+            disksUsage = disksUsage.filter(disk => !disk[0].match(/^zram[0-9]+$/));
```

Comment:
Why not do the filtering first, and then the .map? saves some work. You can also chain these, i.e. no need to re-assign.

Reply:
I'm doing map first because I'd have to remove the respective value from arrays in `this.samples[14]` and `this.samples[15]` if I filtered first. Not sure which approach is better but I can use splice and remove these indexes.

Reply:
Ah right, of course you wouldn't get correct `i` indexes any more. Nevermind then.

Still, you can chain these, looks nicer than reassignment -- plus, you can make it `const`.